website/templates/project/project_base.mako

```diff
@@ -16,11 +16,14 @@
 
 <%include file="modal_show_links.mako"/>
 
-${next.body()}
-
-## % if node['node_type'] == 'project':
-    <%include file="modal_duplicate.mako"/>
-## % endif
+% if node['is_retracted'] == True:
```

Comment:
Load retracted_registration.mako vs modal_duplicate.mako. This *hack* was needed to keep the routing system intact. Basically, this was the least hackish hack I could manage to do without breaking other OSF rules.